pkg/questionModel/confirmation.go

```diff
@@ -0,0 +1,153 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+package questionModel
+
+import (
+	"simple-ec2/pkg/cli"
+	"strings"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+/*
+Confirmation represents a question in which the user confirms a list of configurations. It
+is comprised of two SingleSelectLists: The first is the list of configurations and the second
+is the yes/no option to confirm the launch of the instance.
+*/
+type Confirmation struct {
+	lists      []SingleSelectList
+	choice     string // The chosen option
+	focusIndex int    // The index of the cursor
+	allowEdit  bool   // Whether the configurations list is selectable
+	errorMsg   string // An error message to be presented if a config is selected which cant be reconfigured
+	err        error  // An error caught during the question
+}
+
+// InitializeModel initializes the model based on the passed in question input
+func (c *Confirmation) InitializeModel(input *QuestionInput) {
+	configList := SingleSelectList{}
+	configList.InitializeModel(&QuestionInput{
+		HeaderStrings:  []string{"Configurations", "Values"},
```

Comment:
The header should use singular nouns, to match our other tables.

```suggestion
		HeaderStrings:  []string{"Configuration", "Value"},
```